listenbrainz/webserver/templates/base.html

```diff
@@ -4,60 +4,139 @@
 <html>
 <head>
   {%- block head -%}
-    <meta charset="utf-8" />
-    <title>{% block title %}ListenBrainz{% endblock %}</title>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta charset="utf-8" />
+  <title>{% block title %}ListenBrainz{% endblock %}</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <link rel="icon" sizes="16x16" href="{{ url_for('static', filename='img/favicon-16.png') }}" type="image/png" />
-    <link rel="icon" sizes="256x256" href="{{ url_for('static', filename='img/favicon-256.png') }}" type="image/png" />
-    <link rel="icon" sizes="32x32" href="{{ url_for('static', filename='img/favicon-32.png') }}" type="image/png" />
+  <link rel="icon" sizes="16x16" href="{{ url_for('static', filename='img/favicon-16.png') }}" type="image/png" />
+  <link rel="icon" sizes="256x256" href="{{ url_for('static', filename='img/favicon-256.png') }}" type="image/png" />
+  <link rel="icon" sizes="32x32" href="{{ url_for('static', filename='img/favicon-32.png') }}" type="image/png" />
 
-    {# The css file has a .less extension in the manifest file entry (due to its original name in Webpack entry) #}
-    <link href="{{ get_static_path('main.less') }}" rel="stylesheet" media="screen" />
+  {# The css file has a .less extension in the manifest file entry (due to its original name in Webpack entry) #}
+  <link href="{{ get_static_path('main.less') }}" rel="stylesheet" media="screen" />
   {%- endblock -%}
 </head>
 
 <body>
-  {%- include 'navbar.html' -%}
+{%- include 'navbar.html' -%}
 
-  <div class="container">
+<div class="container">
 
-    {% with messages = get_flashed_messages(with_categories=true) %}
-      {% for category, message in messages %}
-        {{ print_message(message, category) }}
-      {% endfor %}
-    {% endwith %}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+  {% for category, message in messages %}
+  {{ print_message(message, category) }}
+  {% endfor %}
+  {% endwith %}
 
-    {%- block wrapper -%}
-    <div class ="wrapper">
+  {%- block wrapper -%}
+  <div class ="wrapper">
 
-      {%- block content -%}
-      <!-- Page content -->
-      {%- endblock -%}
-    </div>
+    {%- block content -%}
+    <!-- Page content -->
     {%- endblock -%}
+  </div>
+  {%- endblock -%}
+
+  {%- block footer -%}
+  <section class="footer">
```

Comment:
The footer butts up right next to the content of the page*, I think maybe adding a `margin-top: 2em` or something like that might be good.

Currently:
![image](https://user-images.githubusercontent.com/6179856/142858674-1bf77143-22c8-496c-a71c-82af7021a139.png)

Reply:
Done